instance/tests/models/test_instance.py

```diff
@@ -137,9 +137,10 @@ def test_github_admin_username_list_default(self):
         By default, no admin should be configured
         """
         instance = OpenEdXInstanceFactory()
+        instance.reset_ansible_settings()
```

Comment:
@smarnach Rather than adding this to the tests (and having to remember to do this every time an instance is created), is there a way to ensure the settings are correctly set upon the object creation? Maybe through the manager create method and/or the on_pre_save hook, before the initial object creation?